db/migrate/20170601062052_change_column_of_requests.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnOfRequests < ActiveRecord::Migration[5.0]
+  def change
+    change_column :requests, :status, 'integer USING CAST(status AS integer)'
```

Comment:
This is full error message, and when I change column from integer to string, it didn't occur.

```
-- change_column(:requests, :status, :integer)
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled:

PG::DatatypeMismatch: ERROR:  column "status" cannot be cast automatically to type integer
HINT:  You might need to specify "USING status::integer".
: ALTER TABLE "requests" ALTER COLUMN "status" TYPE integer
/Users/hidehiro/code/hidehiro98/airflow_beta/db/migrate/20170602072248_change_column_of_requests3.rb:3:in `change'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/rails:9:in `require'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/rails:9:in `<top (required)>'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/spring:15:in `<top (required)>'
./bin/rails:3:in `load'
./bin/rails:3:in `<main>'
ActiveRecord::StatementInvalid: PG::DatatypeMismatch: ERROR:  column "status" cannot be cast automatically to type integer
HINT:  You might need to specify "USING status::integer".
: ALTER TABLE "requests" ALTER COLUMN "status" TYPE integer
/Users/hidehiro/code/hidehiro98/airflow_beta/db/migrate/20170602072248_change_column_of_requests3.rb:3:in `change'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/rails:9:in `require'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/rails:9:in `<top (required)>'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/spring:15:in `<top (required)>'
./bin/rails:3:in `load'
./bin/rails:3:in `<main>'
PG::DatatypeMismatch: ERROR:  column "status" cannot be cast automatically to type integer
HINT:  You might need to specify "USING status::integer".
/Users/hidehiro/code/hidehiro98/airflow_beta/db/migrate/20170602072248_change_column_of_requests3.rb:3:in `change'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/rails:9:in `require'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/rails:9:in `<top (required)>'
/Users/hidehiro/code/hidehiro98/airflow_beta/bin/spring:15:in `<top (required)>'
./bin/rails:3:in `load'
./bin/rails:3:in `<main>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```